lib/protobuf/nats/client.rb

```diff
@@ -69,7 +120,17 @@ def response_timeout
         end
       end
 
+      def use_subscription_pooling?
+        return @use_subscription_pooling unless @use_subscription_pooling.nil?
+        @use_subscription_pooling = self.class.subscription_pool_size > 0
+      end
+
       def send_request
+        if use_subscription_pooling?
+          available = self.class.subscription_pool.instance_variable_get("@available")
```

Comment:
I think you can just do `self.class.subscription_pool.available`

Reply:
@film42 you can't, that is available in the master branch but has not been released